### PR TITLE
update README

### LINKED ISSUE #8: update README.md file

### PR DESCRIPTION
**Pull request by mariamiah**
## Description
Updates the README file

Fixes #8 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by skizza8 — DISMISSED** [review]

**Comment by AineKiraboMbabazi** [comment]
It looks like the project template comes with a README.md template, was wondering if you got an opportunity to look at it as well

**Comment by mariamiah** [comment]
> It looks like the project template comes with a README.md template, was wondering if you got an opportunity to look at it as well

Yes i did, although modified some parts in it inorder to customize it to our application

File: README.md [hunk 1]
```diff
@@ -1,120 +1,45 @@
...
-  `poetry add --dev <dev-dependency>`
-- Run your project:
-  `poetry run app`
 
+1.  You should have **Python 3.5+** and **git** installed. 
+2.  Inorder to get started, clone this repository using `git clone https://github.com/BuildForSDG/Teachers-Hub-Backend.git`
+3.  Change in the repository directory `cd teachers-hub-backend`
+4.  Set up a virtual environment by running `python3 -m venv <name_of_the_environment>`
+5.  Install requirements by running `pip install -r requirements.txt`
+6.  Navigate to `http://localhost:8000` in the browser to view the api.
```
Comment (daisymacharia) [inline README.md]: You could include the command use to start the flask server as well before navigating to `http://localhost:8000`

**Review by daisymacharia — CHANGES_REQUESTED** [review]

**Review by daisymacharia — APPROVED** [review]
This works for now, we could update it with time if need be.

**Review by AineKiraboMbabazi — APPROVED** [review]

**Review by skizza8 — APPROVED** [review]

**Review by skizza8 — APPROVED** [review]

**Pull request merged**